src/commands/controller.ts

```diff
@@ -97,7 +139,21 @@ export const builder: CommandBuilder<{}, Options> = (yargs) =>
           });
 
           const reg = new Registry({ signer });
-
+          // Check if the passed `name` is valid, otherwise, if it's a table alias
+          try {
+            await globalThis.sqlparser.validateTableName(name);
```

Comment:
i use this try/catch pattern a few times. i feel like i'm overdoing it and wanted to double check if this is the right approach, or if i should let the original error from `validateTableName` bubble up. (i'm not sure what the general rule of thumb is.)

Reply:
For try/catch in general, my preference is to use it as a means to ensure the user gets an error that they can understand.  If during expected running of the cli this results in an error that is confusing, then you can use a try catch and create an error that says something useful.  Here's an example: https://github.com/tablelandnetwork/js-tableland-cli/blob/6b4923d924c0706f9584dc84bc23062f1df3027a/src/utils.ts#L56-L60

As an aside, Carson did some great work configuring the use of [error cause](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause), which is awesome imo.  We probably haven't fully taken advantage of that yet here. That Mozilla article is worth a read if you haven't already.

For the sqlparser specifically, this kind of usage is already setup to provide a nice error for the user.  So the try catch you added here is concealing our standard invalid table name error and creating a one off.  Which is not ideal imo.
Here's the sqlparser source if you want to look at how that works: https://github.com/tablelandnetwork/go-sqlparser/blob/main/cmd/wasm/main.go#L68
As you can see the error says something like, "error validating name: {details on name}".  This is the same error the Validator will get/give, so it's consistent across the whole product set.

RE:  the aliases feature.
It looks like the logic here is:  
 - If you can't validate the name arg, and there is an alias, don't error
 - proceed as normal with whatever name happens to be in the name variable

In this case you're not always validating the uuTableName, which we weren't doing before, but why validate it sometimes?

My thinking is that in general, to enable aliases in the CLI the logic we want to use is:
 - if there's an alias, replace the user specific tableName with the uuTableName
 - proceed as normal with the uuTableName.

In this case, the internals of the command are always the same, the only difference is that if the user wants to map via the aliases file we enable them to do that.

As an example, I'd replace the changes from line 142 to 156 with:
```
if (aliases) {
  const nameMap = await jsonFileAliases(aliases).read();
  const uuName = nameMap[name];
  name = uuName ?? name;
}

await globalThis.sqlparser.validateTableName(name);
```
Or something like that. 
You could probably create a helper that does that logic, and use it in several other places.  That kind of helper is also nice because it's a pattern that helps you to "never mutate state" (a.k.a. never use `let` in js).

Sorry for the novella length response haha, hopefully it's useful 🚀